inference-engine/samples/benchmark_app/benchmark_app.hpp

```diff
@@ -143,6 +143,8 @@ static constexpr char iop_message[] = "Optional. Specifies precision for input a
                                       "                                             Overwrites precision from ip and op options for "
                                       "specified layers.";
 
+static constexpr char float_input_image_blob_message[] = "Optional. Use float input image blob divided by 255. Default value is \"false\".";
```

Comment:
Why do we need this option? `ip` option is not enough?

Reply:
Agree. ip can do it. remove it.